heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/RawDryRunRender.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler.utils;
```

Comment:
let's put all the dry run responses and renderers in `com.twitter.heron.scheduler.dryrun`. Util should almost only ever be used for static utility helper methods, and even then it should be avoided when possible.